src/legacy/core_plugins/kibana/public/visualize/listing/visualize_listing.js

```diff
@@ -30,7 +30,7 @@ import { SavedObjectsClientProvider } from 'ui/saved_objects';
 import { VisualizeListingTable } from './visualize_listing_table';
 import { NewVisModal } from '../wizard/new_vis_modal';
 import { VisualizeConstants } from '../visualize_constants';
-import { setup } from '../../../../visualizations/public/np_ready/public/legacy';
+import { setup as visualizationsSetup } from '../../../../visualizations/public/legacy';
```

Comment:
What do you think of exporting these in their more fully qualified names?  I think getting into the habit of using `as` frequently makes it more difficult to search for references across our code base (albiet typescript should make that easier, I still find it quicker to do ctrl-search)

Reply:
Worth doing, I looked at this and didn't complain since (in theory) these references will eventually be removed.

Reply:
no strong feelings from me on this either way; mostly i want to do something that's consistent across the board so people aren't having to shim things in different ways.

if we do want to use qualified names, i would probably suggest we do that in a subsequent PR so we can address everything (or at least, all App Arch plugins) at once, while also updating the instructions in `MIGRATIONS.md`. but i'm happy to make the vis-specific changes in this PR as a head start.

one small complication here is that in 7.4 we are already giving some instructions on shimming by importing the generic `setup` name, for data at least, so we may need to keep that in place if we want to preserve BWC for any 3rd party devs. (The shim instructions are going out in our "API Changes" blog post so people will already be receiving explicit instructions to use `setup`).

in an ideal world, each service is only requiring devs to update their plugins once during `7.x` for moving to the shim, and then once more to cutover to NP (same contract, just without a shim).

so if we want to make this change, i'd vote we do it sooner rather than later.

WDYT @stacey-gammon @mattkime ?

Reply:
Per offline discussion w/ @mattkime and @stacey-gammon, we'll wait and change these names across the board later, rather than blocking this PR on that update.